pyoxidizer/src/pyrepackager/packaging_rule.rs

```diff
@@ -839,24 +839,18 @@ fn resolve_pip_install_simple(
         pip_args.extend(rule.extra_args.clone().unwrap());
     }
 
-    // TODO send stderr to stdout.
-    let mut cmd = std::process::Command::new(&dist.python_exe)
-        .args(&pip_args)
-        .envs(&extra_envs)
-        .stdout(std::process::Stdio::piped())
-        .spawn()
+    let handler = cmd(&dist.python_exe, &pip_args)
+        .full_env(&extra_envs)
```

Comment:
`.full_env()` will replace the environment variables with the set specified. This is different from what `.envs()` did before, which adds multiple environment variables to the existing set. We need to preserve the old behavior.

Reply:
they don't seem to have implemented this (according to the docs) I think copying the parent's environment variables and chaining them with the extra_envs variables should be satisfactory.